jdaviz/configs/default/plugins/gaussian_smooth/gaussian_smooth.py

```diff
@@ -43,48 +40,30 @@ class GaussianSmooth(TemplateMixin):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.hub.subscribe(self, DataCollectionAddMessage,
-                           handler=self._on_data_updated)
-        self.hub.subscribe(self, DataCollectionDeleteMessage,
-                           handler=self._on_data_updated)
-
-        self._selected_data = None
         if self.config == "cubeviz":
             self.show_modes = True
+            # retrieve the data from the cube, not the collapsed 1d spectrum
+            self.dataset._viewer_refs = ['flux-viewer', 'spectrum-viewer']
```

Comment:
this feels a little hacky - is it worth generalizing more or exposing a more friendly method to do this?